server/resolvers/testPlanReportsResolver.js

```diff
@@ -30,12 +30,7 @@ const testPlanReportsResolver = async (
         attributes: testPlanReportAttributes
     } = retrieveAttributes('testPlanReport', TEST_PLAN_REPORT_ATTRIBUTES, info);
 
-    const { attributes: testPlanRunAttributes } = retrieveAttributes(
-        'draftTestPlanRuns',
-        TEST_PLAN_RUN_ATTRIBUTES,
-        info,
-        true
-    );
+    const testPlanRunAttributes = TEST_PLAN_RUN_ATTRIBUTES;
```

Comment:
We could use the imported variable directly instead of copying a reference here.

Reply:
Hey can you elaborate a little more on this.

Reply:
Since this is no longer a calculation and it also isn't used anywhere else, you can use the `TEST_PLAN_RUN_ATTRIBUTES` imported variable directly at the bottom where this value is passed in `getTestPlanReports`. No need to create a new variable named `testPlanRunAttributes`.

Reply:
I mean really, it looks like you could even pass in null instead and not import`TEST_PLAN_RUN_ATTRIBUTES` in this file as that same import is the [default value for that parameter in `getTestPlanReports`.](https://github.com/w3c/aria-at-app/blob/no-skipped-tests/server/models/services/TestPlanReportService.js#L158) This is a personal preference thing though. I like being explicit and doing my first suggestion.

Reply:
Ok. I went with the first suggestion. I now totally understand and agree with the change.